File: frontend/src/types/triage.ts
export interface TriageResult {
  industry_match: number;
  industry_rationale: string;
  capability_match: number;
  capability_rationale: string;
  case_study_evidence: number;
  case_study_rationale: string;
  recommended_case_studies: string[];
  differentiators: number;
  differentiators_rationale: string;
  total: number;
  band: "Pursue" | "Qualify" | "No Bid";
  capability_gaps: string[];
  key_differentiators: string[];
  risks: string[];
  summary: string;
}

File: backend/app/models/triage.py
from typing import Literal

from pydantic import BaseModel


class TriageResult(BaseModel):
    industry_match: int
    industry_rationale: str
    capability_match: int
    capability_rationale: str
    case_study_evidence: int
    case_study_rationale: str
    recommended_case_studies: list[str]
    differentiators: int
    differentiators_rationale: str
    total: int
    band: Literal["Pursue", "Qualify", "No Bid"]
    capability_gaps: list[str]
    key_differentiators: list[str]
    risks: list[str]
    summary: str
